[2017-01-13 07:58:55,050] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 07:59:04,767] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 08:00:21,381] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 08:00:40,991] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 08:00:50,087] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 08:02:45,268] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 08:02:59,714] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 08:06:29,676] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 08:12:25,116] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} ERROR - Error on request:
Traceback (most recent call last):
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 193, in run_wsgi
    execute(self.server.app)
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 184, in execute
    write(data)
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 152, in write
    self.send_header(key, value)
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 401, in send_header
    self.wfile.write("%s: %s\r\n" % (keyword, value))
IOError: [Errno 32] Broken pipe
[2017-01-13 08:12:44,876] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} ERROR - Error on request:
Traceback (most recent call last):
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 193, in run_wsgi
    execute(self.server.app)
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 184, in execute
    write(data)
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 152, in write
    self.send_header(key, value)
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 401, in send_header
    self.wfile.write("%s: %s\r\n" % (keyword, value))
IOError: [Errno 32] Broken pipe
[2017-01-13 08:30:51,612] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} ERROR - Error on request:
Traceback (most recent call last):
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 193, in run_wsgi
    execute(self.server.app)
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 184, in execute
    write(data)
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 152, in write
    self.send_header(key, value)
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 401, in send_header
    self.wfile.write("%s: %s\r\n" % (keyword, value))
IOError: [Errno 32] Broken pipe
[2017-01-13 08:30:51,617] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} ERROR - Error on request:
Traceback (most recent call last):
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 193, in run_wsgi
    execute(self.server.app)
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 184, in execute
    write(data)
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 152, in write
    self.send_header(key, value)
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 401, in send_header
    self.wfile.write("%s: %s\r\n" % (keyword, value))
IOError: [Errno 32] Broken pipe
[2017-01-13 08:31:03,322] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 08:32:33,995] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 08:32:39,043] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 08:33:57,283] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 08:41:30,777] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 08:41:42,733] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 09:12:25,930] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 09:12:33,364] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 09:16:00,345] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 09:16:08,845] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 09:16:08,934] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 09:16:42,111] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 09:16:55,488] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 09:16:55,646] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 09:17:53,808] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 09:17:53,958] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 09:17:54,172] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 09:18:12,490] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 09:18:42,043] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 09:34:01,856] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 09:34:04,472] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 09:48:45,760] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 09:49:17,254] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 09:49:28,839] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 09:59:43,982] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 09:59:49,817] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 10:00:03,985] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 10:00:06,131] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 10:00:13,751] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 10:00:15,986] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 10:25:49,315] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 10:25:52,066] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 10:26:13,797] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 10:26:26,075] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 10:26:27,738] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 10:26:43,784] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 10:26:45,322] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 10:43:56,297] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 11:10:23,220] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 11:10:29,258] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 11:10:52,240] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 11:11:47,474] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 11:12:05,857] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 11:12:38,573] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 11:12:43,285] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 11:13:10,121] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 11:20:53,446] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 11:22:20,141] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 11:22:20,261] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 11:35:38,753] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 11:35:38,867] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 11:40:42,109] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 11:44:28,686] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 11:58:09,552] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 11:58:09,584] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 12:26:41,913] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 12:26:46,410] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 12:26:47,455] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 12:32:54,254] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 12:32:56,615] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 12:32:58,810] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 12:32:59,961] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 12:33:01,207] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 12:33:02,145] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 12:33:15,968] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 12:33:20,046] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 12:35:14,083] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 12:42:25,504] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 12:44:48,666] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 13:13:44,684] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 13:24:05,689] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 13:32:20,868] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 13:33:46,100] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 13:34:22,929] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 13:34:28,146] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 13:34:51,535] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 13:35:33,899] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 13:36:57,153] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 13:37:06,707] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 13:37:12,975] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 13:37:20,228] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 13:37:42,492] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 13:37:47,472] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 13:40:22,498] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 13:40:52,424] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 13:41:02,574] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 13:41:13,934] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 13:41:50,606] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 13:42:09,352] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 13:42:30,254] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 13:44:15,847] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 13:45:09,901] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 13:45:12,414] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 13:45:12,722] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 13:45:12,881] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 13:45:13,073] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 13:45:13,293] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 13:45:13,495] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-13 13:47:16,141] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 13:47:38,224] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 13:48:49,226] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 13:50:40,552] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 13:51:07,918] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 13:53:44,799] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 13:53:57,464] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 13:56:23,792] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 13:56:46,252] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 14:02:17,007] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 14:02:43,565] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 14:02:57,035] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 14:03:43,857] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 14:04:00,438] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 14:04:15,845] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 14:04:25,076] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 14:06:32,625] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 14:06:56,036] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 14:07:42,066] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 14:10:26,665] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-13 14:11:46,078] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-20 09:39:43,562] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-20 09:40:22,596] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-20 09:42:35,666] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} ERROR - Error on request:
Traceback (most recent call last):
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 193, in run_wsgi
    execute(self.server.app)
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 184, in execute
    write(data)
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 152, in write
    self.send_header(key, value)
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 401, in send_header
    self.wfile.write("%s: %s\r\n" % (keyword, value))
IOError: [Errno 32] Broken pipe
[2017-01-20 09:42:35,671] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} ERROR - Error on request:
Traceback (most recent call last):
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 193, in run_wsgi
    execute(self.server.app)
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 184, in execute
    write(data)
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 152, in write
    self.send_header(key, value)
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 401, in send_header
    self.wfile.write("%s: %s\r\n" % (keyword, value))
IOError: [Errno 32] Broken pipe
[2017-01-20 09:42:35,676] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} ERROR - Error on request:
Traceback (most recent call last):
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 193, in run_wsgi
    execute(self.server.app)
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 184, in execute
    write(data)
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 152, in write
    self.send_header(key, value)
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 401, in send_header
    self.wfile.write("%s: %s\r\n" % (keyword, value))
IOError: [Errno 32] Broken pipe
[2017-01-20 10:57:34,623] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-20 11:24:23,484] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-20 11:24:36,953] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-20 11:25:35,056] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-20 11:47:11,068] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-20 12:37:40,215] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-20 12:42:31,321] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-20 12:53:09,328] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-20 13:20:59,019] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-20 13:48:53,420] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-20 13:49:09,023] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-20 13:49:12,624] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-01-20 13:49:23,472] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} ERROR - Error on request:
Traceback (most recent call last):
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 193, in run_wsgi
    execute(self.server.app)
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 183, in execute
    for data in application_iter:
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/debug/__init__.py", line 322, in debug_application
    traceback.log(environ['wsgi.errors'])
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/debug/tbtools.py", line 292, in log
    logfile.write(tb)
IOError: [Errno 32] Broken pipe
[2017-01-20 13:49:38,383] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} ERROR - Error on request:
Traceback (most recent call last):
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 193, in run_wsgi
    execute(self.server.app)
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 183, in execute
    for data in application_iter:
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/debug/__init__.py", line 322, in debug_application
    traceback.log(environ['wsgi.errors'])
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/debug/tbtools.py", line 292, in log
    logfile.write(tb)
IOError: [Errno 32] Broken pipe
[2017-01-20 13:49:45,271] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} ERROR - Error on request:
Traceback (most recent call last):
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 193, in run_wsgi
    execute(self.server.app)
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 183, in execute
    for data in application_iter:
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/debug/__init__.py", line 322, in debug_application
    traceback.log(environ['wsgi.errors'])
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/debug/tbtools.py", line 292, in log
    logfile.write(tb)
IOError: [Errno 32] Broken pipe
[2017-01-20 13:49:46,712] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} ERROR - Error on request:
Traceback (most recent call last):
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 193, in run_wsgi
    execute(self.server.app)
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 183, in execute
    for data in application_iter:
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/debug/__init__.py", line 322, in debug_application
    traceback.log(environ['wsgi.errors'])
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/debug/tbtools.py", line 292, in log
    logfile.write(tb)
IOError: [Errno 32] Broken pipe
[2017-01-20 13:49:46,998] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} ERROR - Error on request:
Traceback (most recent call last):
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 193, in run_wsgi
    execute(self.server.app)
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 183, in execute
    for data in application_iter:
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/debug/__init__.py", line 322, in debug_application
    traceback.log(environ['wsgi.errors'])
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/debug/tbtools.py", line 292, in log
    logfile.write(tb)
IOError: [Errno 32] Broken pipe
[2017-01-20 13:49:47,277] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} ERROR - Error on request:
Traceback (most recent call last):
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 193, in run_wsgi
    execute(self.server.app)
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 183, in execute
    for data in application_iter:
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/debug/__init__.py", line 322, in debug_application
    traceback.log(environ['wsgi.errors'])
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/debug/tbtools.py", line 292, in log
    logfile.write(tb)
IOError: [Errno 32] Broken pipe
[2017-01-20 13:49:47,524] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} ERROR - Error on request:
Traceback (most recent call last):
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 193, in run_wsgi
    execute(self.server.app)
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 183, in execute
    for data in application_iter:
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/debug/__init__.py", line 322, in debug_application
    traceback.log(environ['wsgi.errors'])
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/debug/tbtools.py", line 292, in log
    logfile.write(tb)
IOError: [Errno 32] Broken pipe
[2017-01-20 13:49:47,809] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} ERROR - Error on request:
Traceback (most recent call last):
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 193, in run_wsgi
    execute(self.server.app)
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 183, in execute
    for data in application_iter:
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/debug/__init__.py", line 322, in debug_application
    traceback.log(environ['wsgi.errors'])
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/debug/tbtools.py", line 292, in log
    logfile.write(tb)
IOError: [Errno 32] Broken pipe
[2017-01-20 13:49:48,413] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} ERROR - Error on request:
Traceback (most recent call last):
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 193, in run_wsgi
    execute(self.server.app)
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 183, in execute
    for data in application_iter:
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/debug/__init__.py", line 322, in debug_application
    traceback.log(environ['wsgi.errors'])
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/debug/tbtools.py", line 292, in log
    logfile.write(tb)
IOError: [Errno 32] Broken pipe
[2017-01-20 13:49:48,652] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} ERROR - Error on request:
Traceback (most recent call last):
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 193, in run_wsgi
    execute(self.server.app)
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 183, in execute
    for data in application_iter:
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/debug/__init__.py", line 322, in debug_application
    traceback.log(environ['wsgi.errors'])
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/debug/tbtools.py", line 292, in log
    logfile.write(tb)
IOError: [Errno 32] Broken pipe
[2017-01-20 13:49:56,639] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} ERROR - Error on request:
Traceback (most recent call last):
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 193, in run_wsgi
    execute(self.server.app)
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/serving.py", line 183, in execute
    for data in application_iter:
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/debug/__init__.py", line 322, in debug_application
    traceback.log(environ['wsgi.errors'])
  File "/home/ucitel/.env/lib/python2.7/site-packages/werkzeug/debug/tbtools.py", line 292, in log
    logfile.write(tb)
IOError: [Errno 32] Broken pipe
[2017-01-20 13:50:30,472] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-01-20 14:04:59,174] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 08:12:12,977] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 08:13:32,156] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-02-24 08:13:36,055] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-02-24 08:19:13,161] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-02-24 08:19:15,074] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-02-24 08:19:32,213] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-02-24 08:19:32,277] {/home/ucitel/Flask-Bones/app/__init__.py:85} INFO - 404: Not Found
[2017-02-24 08:20:10,446] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 08:21:01,658] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 08:22:23,140] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 08:22:41,951] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 08:23:44,600] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 08:24:39,705] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 08:25:31,170] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 08:25:49,531] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 08:26:00,931] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 08:26:13,532] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 08:26:54,740] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 08:27:11,640] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 08:27:33,347] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 08:27:54,543] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 08:28:03,813] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 08:28:10,871] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 08:29:31,970] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 08:29:46,295] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 08:29:50,575] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 08:29:54,703] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 08:30:08,896] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 09:09:47,605] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 09:10:34,258] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 09:14:31,700] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 09:15:16,510] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 09:15:23,775] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 09:16:29,326] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 09:16:43,648] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-02-24 09:18:23,289] {/home/ucitel/.env/local/lib/python2.7/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-05-29 08:49:25,634] {/home/ucitel/.env/lib/python3.4/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-05-29 08:49:53,871] {/home/ucitel/.env/lib/python3.4/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-05-29 08:55:43,788] {/home/ucitel/.env/lib/python3.4/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-05-29 09:29:07,122] {/home/ucitel/.env/lib/python3.4/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-05-29 09:34:22,044] {/home/ucitel/.env/lib/python3.4/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-05-29 09:52:21,015] {/home/ucitel/.env/lib/python3.4/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-05-29 09:56:41,949] {/home/ucitel/.env/lib/python3.4/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-05-29 09:59:34,330] {/home/ucitel/.env/lib/python3.4/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-05-29 10:01:48,124] {/home/ucitel/.env/lib/python3.4/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-05-29 10:02:00,386] {/home/ucitel/.env/lib/python3.4/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-05-29 10:03:16,618] {/home/ucitel/.env/lib/python3.4/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-05-29 11:04:42,288] {/home/ucitel/.env/lib/python3.4/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-05-29 11:21:16,618] {/home/ucitel/.env/lib/python3.4/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
[2017-05-29 11:23:36,268] {/home/ucitel/.env/lib/python3.4/site-packages/werkzeug/_internal.py:87} WARNING -  * Debugger is active!
